config/redirects

```diff
@@ -59,4 +59,37 @@ raw: ${prefix}/stable -> ${base}/current/
 [*-v1.19]: ${prefix}/${version}/tutorial/example-data/ -> ${base}/v1.x/
 [*-v1.19]: ${prefix}/${version}/tutorial/modeling-bson-data/ -> ${base}/v1.x/data-formats/modeling-bson-data/
 [*-v1.19]: ${prefix}/${version}/tutorial/stable-api/ -> ${base}/v1.x/stable-api/
-[*-v1.19]: ${prefix}/${version}/tutorial/aws-lambda/ -> ${base}/v1.x/aws-lambda/
\ No newline at end of file
+[*-v1.19]: ${prefix}/${version}/tutorial/aws-lambda/ -> ${base}/v1.x/aws-lambda/
+
+# comprehensive coverage redirects
+[v2.x-*]: ${prefix}/${version}/aggregation/atlas-search/ -> ${base}/${version}/atlas-search/
```

Comment:
Q: are we going to backport the TOC changes as we've done with the other languages? in that case, should you redirect all versions?